tests/test_main.py

```diff
@@ -397,6 +398,86 @@ class Model(BaseModel):
     ]
 
 
+def test_const_list():
+    class SubModel(BaseModel):
+        b: int
+
+    class Model(BaseModel):
+        a: List[SubModel] = Schema([SubModel(b=1), SubModel(b=2), SubModel(b=3)], const=True)
+        b: List[SubModel] = Schema([{'b': 4}, {'b': 5}, {'b': 6}], const=True)
+
+    m = Model()
+    assert m.a == [SubModel(b=1), SubModel(b=2), SubModel(b=3)]
+    assert m.b == [SubModel(b=4), SubModel(b=5), SubModel(b=6)]
```

Comment:
would it also help to add `m.schema() == {...}` here?

Reply:
Added, So this is interesting:

It seems that adding SubModel instances as default breaks in a lot of ways: Also the schema is now not json serializable, similar to the error as per #793 .

It might be a good idea to just document or add a warning of some sorts for that case and consider that unsupported. Of course not all use cases need json serializability and some use cases might want to set the default using a SubModel instance.

Reply:
please create an issue about this.